.github/workflows/ccpp.yml

```diff
@@ -47,7 +47,7 @@ jobs:
            cd gccrs-build;
            ../configure \
                --enable-languages=rust \
-               --disable-bootstrap \
+               --enable-bootstrap \
```

Comment:
This needs to stay as `--disable-bootstrap`

Reply:
Yeah, I resolve this, thanks